### ISSUE TITLE
MITxPro: system name is wrong in Studio emails

### ISSUE THREAD

**Issue by pdpinch** [mitodl/salt-ops#755]
Not a critical issue (at least, not yet): the subject line and from address in studio-generated emails still reference "MITx Residential" 

<img width="699" alt="image" src="https://user-images.githubusercontent.com/430126/53302585-8191a380-382d-11e9-9784-d88f92ecaeae.png">

**Comment by markbreedlove** [comment]
Reopening. Will close when I've deployed the change.

**Comment by markbreedlove** [comment]
I've deployed to MITxPro QA. Does it look OK?

**Comment by pdpinch** [comment]
Sorry, still not working (see below). You can generate this email by logging in to https://studio-xpro-qa.mitx.mit.edu/ and going in to any course and exporting it, e.g. https://studio-xpro-qa.mitx.mit.edu/export/course-v1:TestX+LASERxB1+1T2019

This is not a critical issue, but I'm concerned it's a sign of other configuration issues. Can we just search the configuration settings for xpro for mentions of "Residential"?

![image](https://user-images.githubusercontent.com/430126/57386950-73dc9080-7183-11e9-994c-56f13982c603.png)

FWIW, this particular email is generated by studio/CMS.

**Comment by markbreedlove** [comment]
I've learned from Tobias that I needed to restart supervisord on the worker instances. Having done that, I get the correct name in the subject line:
<img width="702" alt="Screen Shot 2019-05-08 at 4 50 30 PM" src="https://user-images.githubusercontent.com/5550828/57407693-c92f9680-71b1-11e9-9468-9bb0138c36a8.png">

On another note, though, the from address says `mitx-support`. Should it say MITxPro? (Though that may be something for a separate ticket.)
